src/components/TopBar.js

```diff
@@ -1,62 +1,65 @@
 import React from 'react';
 import { makeStyles } from '@material-ui/core/styles';
-import { AppBar, Toolbar, Button, Typography, ThemeProvider, Box } from '@material-ui/core';
-import { createMuiTheme, StylesProvider } from '@material-ui/core/styles';
+import { AppBar, Toolbar, Button, Typography, Box } from '@material-ui/core';
+import { StylesProvider } from '@material-ui/core/styles';
+import { ExitToApp } from '@material-ui/icons';
 import '../style/TopBarStyling.css';
+import logo from '../style/FSH-online-logo.png';
 
 const useStyles = makeStyles((theme) => ({
   root: {
-    background: '#2c4f85',
+    background: '#065171',
```

Comment:
This change makes it so that we no longer match the color of FSHSchool, which I thought was something we wanted for the top bar. I don't much mind, just wanted to make sure that was a conscious choice to move away from the matching FSHSchool.

Reply:
Yeah I consciously changed it to the color Noranda had in her design. The new color seems to be somewhere between the blue that's on the FSH School [landing page](https://fshschool.org/) (which was the old blue color here) and the blue of the top bar on the [Documentation pages](https://fshschool.org/docs/). I don't think I know enough about colors to say which way is better. Do you have any thoughts?

Reply:
I matched them, and used the blue directly from the [documentation page](https://fshschool.org/docs/), so it's still different than it used to be, but I think it makes more sense since the top bars now match. Let me know if you prefer the original blue though.

Reply:
+1 on top bars matching.  It looks great.